src/main.js

```diff
@@ -79,10 +89,10 @@ const setCORSHeaders = (request, response, next) => {
     });
     return response.sendStatus(204);
   }
-	else {
-		response.set({'Access-Control-Allow-Origin': '*'});
-		next();
-	}
+  else {
+    response.set({ 'Access-Control-Allow-Origin': '*' });
```

Comment:
Yeah.. probably.. Or find a different way to set this that conforms to their api.